docs/ossm/ossm2-migration/multi-tenancy/README.md

```diff
@@ -0,0 +1,285 @@
+# OSSM 2 --> 3 Multi-tenancy Migration guide
+
+This guide is for users who are currently running `MultiTenant` OSSM 2.x migrating to OSSM 3.x. You should first read [this document comparing OSSM 2 vs. OSSM 3](../../ossm2-vs-ossm3.md) to familiarize yourself with the concepts between the two versions and the differences in manging `MultiTenant` workloads. Specifically the [Scoping the Mesh section](../../ossm2-vs-ossm3.md#scoping-of-the-mesh-discovery-selectors-and-labels-replace-servicemeshmemberroll-and-servicemeshmember) is important for migrating from OSSM 2 to OSSM 3.
+
+## Migrating OSSM 2 Multi-Tenant to OSSM 3
+
+### Prerequisites
+
+- OSSM2 operator is installed
+- OSSM3 operator is installed
+- `IstioCNI` is installed
+- `istioctl` is installed
+- MultiTenant SMCP instance
+
+### Procedure
+
+In this example, we'll be using the [bookinfo demo](https://raw.githubusercontent.com/Maistra/istio/maistra-2.6/samples/bookinfo/platform/kube/bookinfo.yaml) but you can follow these same steps with your own workloads.
+
+Before you begin, please ensure that you have completed all the steps in the [pre-migration checklist](../README.md#pre-migration-checklist).
+
+<!-- Developer instructions for testing with a fresh 2.6 install.
+
+1. Create SMCP namespace
+
+```sh
+oc create ns tenant-a
+```
+
+1. Create SMCP
+
+   ```yaml
+   apiVersion: maistra.io/v2
+   kind: ServiceMeshControlPlane
+   metadata:
+     name: basic
+     namespace: tenant-a
```

Comment:
For consistency (with the OSSM 2.x doc examples), should we choose a namespace similar to `istio-system` for the location of SMCP? Maybe `istio-system-a`? OR `istio-system-tenant-a` (maybe a bit too verbose..). 

Scanning the instructions quickly, at first I thought `tenant-a` referred to the data plane namespace (`bookinfo` in this case), as I think most users would consider the app namespace to be the real tenant.

Reply:
`istio-system-tenant-a` sounds fine. To be honest I wasn't sure how people usually named these.